WordPress/Classes/ViewRelated/NUX/LoginEpilogueAnimator.swift

```diff
@@ -0,0 +1,80 @@
+import UIKit
+
+final class LoginEpilogueAnimator: NSObject, UIViewControllerAnimatedTransitioning {
+
+    private let loginAnimationDuration = 0.3
+    private let quickStartAnimationDuration = 0.3
```

Comment:
These two can also go to a `Constants`. :)

Reply:
Sounds good, I moved these to a constant enum.